daemon/api_users.go

```diff
@@ -276,7 +277,20 @@ func postCreateUser(c *Command, r *http.Request, user *auth.UserState) Response
 	if err := decoder.Decode(&createData); err != nil {
 		return BadRequest("cannot decode create-user data from request body: %v", err)
 	}
-	return createUser(c, createData)
+
+	rsp := createUser(c, createData)
```

Comment:
a way of avoiding the below nasty code would be to give `createUser` a flag to say "do the old thing". Not sure which is better.